qa/L0_perf_deeprecommender/test.sh

```diff
@@ -0,0 +1,231 @@
+MODEL="deeprecommender"
+
+rm -f *.log *.serverlog *.csv *.metrics *.tjson *.json
+
+#
+# Test minimum latency
+#
+STATIC_BATCH=1
+DYNAMIC_BATCH=1
+INSTANCE_CNT=1
+
+# Copy TF Model
+rm -fr tfmodels && mkdir -p tfmodels && \
+    cp -r $REPODIR/tf_model_store2/deep_recommender tfmodels/deeprecommender_graphdef && \
+    (cd tfmodels/deeprecommender_graphdef && \
+        sed -i "s/^name:.*/name: \"deeprecommender_graphdef\"/" config.pbtxt && \
+        sed -i "s/^max_batch_size:.*/max_batch_size: ${STATIC_BATCH}/" config.pbtxt && \
+        echo "instance_group [ { count: ${INSTANCE_CNT} }]" >> config.pbtxt)
+
+# Create the TensorRT plan from TF
+rm -fr tensorrt_models && mkdir tensorrt_models
+    cp -r $REPODIR/tf_model_store2/deep_recommender tensorrt_models/deeprecommender_plan && \
+    (cd tensorrt_models/deeprecommender_plan && \
+        sed -i "s/^name:.*/name: \"deeprecommender_plan\"/" config.pbtxt && \
+        sed -i "s/^max_batch_size:.*/max_batch_size: ${STATIC_BATCH}/" config.pbtxt && \
+        sed -i "s/tensorflow_graphdef/tensorrt_plan/" config.pbtxt && \
+        sed -i "s/\[17736\]/\[17736,1,1\]/" config.pbtxt && \
+        echo "instance_group [ { count: ${INSTANCE_CNT} }]" >> config.pbtxt)
+
+$TRTEXEC --uff=tfmodels/deeprecommender_graphdef/deep_recommender.uff \
+         --uffInput=Placeholder,1,1,17736\
+         --batch=${STATIC_BATCH} --output=fc5/Relu --verbose \
+         --saveEngine=tensorrt_models/deeprecommender_plan/0/model.plan
+
+# Create the TFTRT plan from TF
+rm -fr tftrt_models && mkdir tftrt_models
+    cp -r $REPODIR/tf_model_store2/deep_recommender tftrt_models/deeprecommender_graphdef_trt && \
+    (cd tftrt_models/deeprecommender_graphdef_trt && \
+        sed -i "s/^name:.*/name: \"deeprecommender_graphdef_trt\"/" config.pbtxt && \
+        sed -i "s/^max_batch_size:.*/max_batch_size: ${STATIC_BATCH}/" config.pbtxt && \
+        echo "optimization { execution_accelerators {" >> config.pbtxt
+        echo "gpu_execution_accelerator : [ {" >> config.pbtxt
+        echo "name : \"tensorrt\" " >> config.pbtxt
+        echo "} ]" >> config.pbtxt
+        echo "}}" >> config.pbtxt
+        echo "instance_group [ { count: ${INSTANCE_CNT} }]" >> config.pbtxt)
+
+# Tests with each model
+for FRAMEWORK in graphdef plan graphdef_trt; do
+    MODEL_NAME=${MODEL}_${FRAMEWORK}
+    if [ "$FRAMEWORK" == "graphdef" ]; then
+        REPO=`pwd`/tfmodels
+    elif [ "$FRAMEWORK" == "plan" ]; then
+        REPO=`pwd`/tensorrt_models
+    else
+        REPO=`pwd`/tftrt_models
+    fi
+    FRAMEWORK=$(echo ${FRAMEWORK} | cut -d '_' -f 1)
+    MODEL_NAME=${MODEL_NAME} \
+            MODEL_FRAMEWORK=${FRAMEWORK} \
+            MODEL_PATH="$REPO/${MODEL_NAME}" \
+            STATIC_BATCH_SIZES=${STATIC_BATCH} \
+            DYNAMIC_BATCH_SIZES=${DYNAMIC_BATCH} \
+            INSTANCE_COUNTS=${INSTANCE_CNT} \
+            bash -x run_test.sh
+done
+
+#
+# Test large static batch = 128 w/ 2 instances
+#
+STATIC_BATCH=128
```

Comment:
I think you can likely use a much larger size here since recommender is a small model. Can you try 512 and 1024 and see if memory usage is ok.

Reply:
Passed for STATIC_BATCH=512 and 1024, setting to 1024